README.md

```diff
@@ -1,27 +1,57 @@
-
 ## LSST Data Processing Explorer
-[![Travis Build Status](https://travis-ci.com/Quansight/lsst_dashboard.svg?branch=master)](https://travis-ci.com/Quansight/lsst_dashboard)
 
-### Getting Started...
+Here are end-to-end instructions of how to currently set up and run the in-development QA Dashboard on a dataset of your choosing.  N.B. the dataset must have had the `pipe_analysis` scripts and `visitMatch.py` run on them already.
+
+### Setting up the stack
 
-1. Download and Extract sample dataset from here:
-https://quansight.nyc3.digitaloceanspaces.com/datasets/RC2_w18.tar.gz
+After sourcing the current stack, set up the `tickets/DM-21335` branches of `qa_explorer` and `obs_base`.  Run `scons` on `qa_explorer` in order to make the `prepareQADashboard.py` command-line task available.
 
-**OR**
-Simply use the `download_sample_data.sh` to download smaller subsets.
+### Install this package and various dependencies
 
-2. Setup and activate conda environment with Python 3.7 and PyViz stack
+After the stack has been set up, install this package and a number of dependencies:
 ```
-conda env create -f environment.yml
-conda activate lsst-panel
-python setup.py develop
+git clone https://github.com/timothydmorton/lsst_dashboard
```

Comment:
Is linking to your `timothydmorton` repo intentional? i.e. do we want to make that the master repo? If so we may want to transfer the repo so that issues and stuff are still available.

Reply:
setup.py still refers to he current repo.

Reply:
Ah, thanks for catching that- I had put this up on my fork originally.  I think we should transfer at some point, but probably to the lsst-dm organization.  I'll change this to point to the current master repo for now.